### PR TITLE
Add singularity-permanent-cache recipe.

### PR DESCRIPTION
**Pull request by rhpvorderman**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by DavyCats** [comment]
Just commenting to confirm that I'm willing to be a maintainer.

File: recipes/singularity-permanent-cache/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "singularity-permanent-cache" %}
+{% set version = "1.0.0a0" %}
```
Comment (scopatz) [inline recipes/singularity-permanent-cache/meta.yaml:2]: Please remove a0

Comment (rhpvorderman) [reply recipes/singularity-permanent-cache/meta.yaml:2]: This is an alpha version.

**Comment by Redmar-van-den-Berg** [comment]
I'm also willing to be a maintainer

File: recipes/singularity-permanent-cache/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/singularity-permanent-cache/meta.yaml:2]: Is there a fully released version we can package. We tend not to package alpha versions, or other prereleases. There is a CFEP that you can check for instructions on how to do that, though

Comment (rhpvorderman) [reply recipes/singularity-permanent-cache/meta.yaml:2]: @scopatz. Thanks! We will encourage users to install via PYPI and will open this PR to release 1.0.0 as soon as this RC proves to be stable.

**Pull request closed without merging**